ui/app/serializers/transform.js

```diff
@@ -2,7 +2,7 @@ import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
   normalizeResponse(store, primaryModelClass, payload, id, requestType) {
-    if (payload.data.masking_character) {
```

Comment:
Before this change, attempting to save a new template (or any of the other items) would throw this mysterious error in the console: 
```
Uncaught (in promise) TypeError: e is undefined
    persist transform-edit-base.js:87
    invokeCallback rsvp.js:495
    publish rsvp.js:478
    publishRejection rsvp.js:414
    <anonymous> Ember
```

Reply:
That looks like a fun error to track down 😬